docs/resources/compute_interface_attach_v2.md

```diff
@@ -6,130 +6,106 @@ page_title: "flexibleengine_compute_interface_attach_v2"
 
 ```hcl
-resource "flexibleengine_networking_port_v2" "example_ports" {
-  count          = 2
-  name           = format("port-%02d", count.index + 1)
-  network_id     = flexibleengine_vpc_subnet_v1.example_subnet.id
-  admin_state_up = "true"
-}
+variable "security_group_id" {}
 
-resource "flexibleengine_compute_interface_attach_v2" "example_attachments" {
-  count          = 2
-  instance_id = flexibleengine_compute_instance_v2.example_instance.id
-  port_id     = flexibleengine_networking_port_v2.example_ports.*.id[count.index]
+data "flexibleengine_vpc_subnet_v1" "mynet" {
+  name = "subnet-default"
 }
-```
-
-Note that the above example will not guarantee that the ports are attached in
-a deterministic manner. The ports will be attached in a seemingly random
-order.
 
-If you want to ensure that the ports are attached in a given order, create
-explicit dependencies between the ports, such as:
-
-```hcl
-resource "flexibleengine_networking_port_v2" "example_ports" {
-  count          = 2
-  name           = format("port-%02d", count.index + 1)
-  network_id     = flexibleengine_vpc_subnet_v1.example_subnet.id
-  admin_state_up = "true"
+data "flexibleengine_networking_port" "myport" {
+  network_id = data.flexibleengine_vpc_subnet_v1.mynet.id
+  fixed_ip   = "192.168.0.100"
 }
 
-resource "flexibleengine_compute_interface_attach_v2" "example_interface_attach_1" {
-  instance_id = flexibleengine_compute_instance_v2.instance_1.id
-  port_id     = flexibleengine_networking_port_v2.example_ports.*.id[0]
+resource "flexibleengine_compute_instance_v2" "myinstance" {
+  name              = "instance"
+  image_id          = "ad091b52-742f-469e-8f3c-fd81cadf0743"
+  flavor_id         = "s3.large.2"
+  key_pair          = "my_key_pair_name"
+  security_groups   = [flexibleengine_networking_secgroup_v2.test.name]
+  availability_zone = "eu-west-0a"
+
+  network {
+    uuid = flexibleengine_vpc_subnet_v1.mynet.id
+  }
 }
 
-resource "flexibleengine_compute_interface_attach_v2" "example_interface_attach_2" {
-  instance_id = flexibleengine_compute_instance_v2.instance_1.id
-  port_id     = flexibleengine_networking_port_v2.example_ports.*.id[1]
+resource "flexibleengine_compute_interface_attach_v2" "attached" {
+  instance_id = flexibleengine_compute_instance_v2.myinstance.id
+  port_id     = data.flexibleengine_networking_port.myport.id
 }
 ```
 
 ## Argument Reference
 
 The following arguments are supported:
 
-* `region` - (Optional, String, ForceNew) The region in which to create the interface attachment.
-  If omitted, the `region` argument of the provider is used. Changing this creates a new attachment.
+* `region` - (Optional, String, ForceNew) The region in which to create the network interface attache resource. If
+  omitted, the provider-level region will be used. Changing this creates a new network interface attache resource.
 
 * `instance_id` - (Required, String, ForceNew) The ID of the Instance to attach the Port or Network to.
```

Comment:
lack
```md
Changing this creates a new network interface attache resource.
```

check if other fields is `ForceNew` and if have `Changing this...` in this document.

Reply:
done